src/pybind/mgr/dashboard/controllers/mgr_modules.py

```diff
@@ -21,14 +21,19 @@ def list(self):
         """
         result = []
         mgr_map = mgr.get('mgr_map')
+        # Get the always on Manager modules.
+        always_on_modules = mgr_map['always_on_modules'][mgr.release_name]
+        # Collect all available Manager modules.
         for module_config in mgr_map['available_modules']:
             if module_config['name'] not in self.ignore_modules:
                 result.append({
                     'name': module_config['name'],
-                    'enabled': False,
+                    'enabled': module_config['name'] in always_on_modules,
```

Comment:
nit, can we consolidate the code a little bit? like
```py
name = module_config['name']
if name not in self.ignore_modules:
  always_on = name in always_on_modules
  enabled = name in mgr_map['modules'] or always_on
  result.append({
      'name': name,
      'enabled': enabled,
      'always_on': enabled,
      'options': self._convert_module_options(
          module_config['module_options'])
  })
```

Reply:
Done